### PR TITLE
add 4 missing system variables

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
add 4 missing system variables

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9875 https://github.com/pingcap/docs/pull/9779 https://github.com/pingcap/docs/pull/9981
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1603,6 +1603,14 @@ mysql> desc select count(distinct a) from test.t;
 - 这个变量用来设置将 Limit 和 TopN 算子下推到 TiKV 的阈值。
 - 如果 Limit 或者 TopN 的取值小于等于这个阈值，则 Limit 和 TopN 算子会被强制下推到 TiKV。该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
 
+### `tidb_opt_mpp_outer_join_fixed_build_side` <span class="version-mark">从 v5.1.0 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:1606]: @fixdb PTAL

File: system-variables.md [hunk 2]
```diff
@@ -1635,6 +1643,14 @@ explain select * from t where age=5;
 3 rows in set (0.00 sec)
 ```
 
+### `tidb_opt_projection_push_down` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：布尔值
+- 默认值：`OFF`
+- 指定是否允许优化器将 `Projection` 算子下推到 TiKV 或者 TiFlash 协处理器。
```
Comment (TomShawn) [inline system-variables.md:1652]: @fixdb PTAL

File: system-variables.md [hunk 3]
```diff
@@ -1767,6 +1783,14 @@ explain select * from t where age=5;
 - 这个变量用来控制优化器是否可以将包含 null 的等值条件作为前缀条件来访问索引。
 - 该变量默认开启。开启后，该变量可以使优化器减少需要访问的索引数据量，从而提高查询的执行速度。例如，在有多列索引 `index(a, b)` 且查询条件为 `a<=>null and b=1` 的情况下，优化器可以同时使用查询条件中的 `a<=>null` 和 `b=1` 进行索引访问。如果关闭该变量，因为 `a<=>null and b=1` 包含 null 的等值条件，优化器不会使用 `b=1` 进行索引访问。
 
+### `tidb_remove_orderby_in_subquery` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔值
+- 默认值：`OFF`
+- 指定是否在子查询中移除 `ORDER BY` 子句。
```
Comment (TomShawn) [inline system-variables.md:1792]: @fixdb PTAL

File: system-variables.md [hunk 4]
```diff
@@ -1603,6 +1603,14 @@ mysql> desc select count(distinct a) from test.t;
 - 这个变量用来设置将 Limit 和 TopN 算子下推到 TiKV 的阈值。
 - 如果 Limit 或者 TopN 的取值小于等于这个阈值，则 Limit 和 TopN 算子会被强制下推到 TiKV。该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
 
+### `tidb_opt_mpp_outer_join_fixed_build_side` <span class="version-mark">从 v5.1.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔值
+- 默认值：`ON`
+- 指定是否允许在 TiFlash 的外部联结中使用外部表作为构建端。
```
Comment (Oreoxmt) [inline system-variables.md:1612]:
```suggestion
- 指定是否允许在 TiFlash 的外连接中使用外部表作为构建端。
```

Comment (fixdb) [reply system-variables.md:1612]: @windtalker Please take a look. I don't think the doc is correct here.
我的理解是：
当这个值设为ON的时候，Left Join始终使用内表作为build端，Right Join始终使用外表作为build端。
当为OFF的时候，外连接可以灵活选择任意一边表作为build端。

这么理解对不对？

File: system-variables.md (hunk 2 above)
Comment (fixdb) [reply system-variables.md:1652]: `协处理器`就不需要了

File: system-variables.md (hunk 4 above)
Comment (windtalker) [reply system-variables.md:1612]: 是的，就是这样的

Comment (TomShawn) [reply system-variables.md:1612]: 英文写的是“Specifies whether to allow using outer table as the build side in outer join for TiFlash.”。请检查一下英文是否正确。如果不正确，辛苦再改下英文，谢谢🙏

File: system-variables.md (hunk 2 above)
Comment (TomShawn) [reply system-variables.md:1652]:
```suggestion
- 指定是否允许优化器将 `Projection` 算子下推到 TiKV 或者 TiFlash。
```

File: system-variables.md (hunk 4 above)
Comment (TomShawn) [reply system-variables.md:1612]:
```suggestion
- 当该变量值为 `ON` 时，左连接始终使用内表作为构建端，右连接始终使用外表作为构建端。将该变量值设为 `OFF` 后，外连接可以灵活选择任意一边表作为构建端。
```

**Review by fixdb — APPROVED** [review]
+1

File: system-variables.md [hunk 5]
```diff
@@ -2003,6 +2027,18 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 单位：字节
 - 这个变量用于限制单个[临时表](/temporary-tables.md)的最大大小，临时表超出该大小后报错。
 
+### `tidb_track_aggregate_memory_usage`
+
+- 作用域：SESSION ｜ GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔值
+- 默认值：`true`
```
Comment (fixdb) [inline system-variables.md:2035]: 是不是应该为`ON`?

Comment (TomShawn) [reply system-variables.md:2035]:
```suggestion
- 默认值：`ON`
```

**Pull request merged**